example/build.gradle

```diff
@@ -1,8 +1,8 @@
 description = 'Hibernate Reactive Example'
 
 dependencies {
-    compile project(':hibernate-reactive-core')
-    compile 'org.hibernate:hibernate-core'
+    implementation project(':hibernate-reactive-core')
+    implementation 'org.hibernate:hibernate-core'
```

Comment:
@Sanne can't we just remove this dependency altogether?

Reply:
ah yes, let me fix that..   I didn't focus on the example other than removing the deprecation warnings I was getting :)